.teamcity/settings.kts

```diff
@@ -816,5 +816,13 @@ object WpDesktop_DesktopE2ETests : BuildType({
 		}
 	}
 
+	triggers {
+		vcs {
+			branchFilter = """
```

Comment:
Since we'll continue using Circle only for merges to master (see #47475), should we exclude master/trunk from TC since that would be effectively duplicating efforts?

Reply:
IMO running tests for master is the best way to track stats.

For example, if we want to track number of failed tests over a period of time, failing tests for branches will include flakey tests and legit errors (the developer broke something). Being able to track them only for master gives us a better signal.

Reply:
That makes sense. In that case I'll go ahead and disable the Circle linux job from running in the master branch (see [here](https://github.com/Automattic/wp-calypso/pull/47475/commits/59a06848cd74cd14bea4834d4a9ee0596fc5e1ef)). So in master, we'll have Linux (TeamCity), and Mac + Windows (Circle).